htdocs/knowledgemanagement/knowledgerecord_card.php

```diff
@@ -225,6 +239,13 @@
 	// Common attributes
 	include DOL_DOCUMENT_ROOT.'/core/tpl/commonfields_edit.tpl.php';
 
+	//Language of question/response
```

Comment:
Same.
Can you include this into the commonfields_edit.tpl.php so every module that use a 'lang' field will benefit the language selector/beautifier.